pkg/operator/mon/health.go

```diff
@@ -118,6 +124,27 @@ func (c *Cluster) checkHealth() error {
 		}
 	}
 
+	if len(monsNotFound) > 0 {
+		for _, mon := range monsNotFound {
+			// If not yet set, add the current time, for the timeout
+			// calculation, to the list
+			// when the mon appears in the mon map the timeout will be deleted
+			if _, ok := c.monTimeoutList[mon.Name]; !ok {
```

Comment:
if the mon isn't found in the monmap, i wouldn't expect it to ever come back. Could we fail it over immediately instead of waiting for the timeout?